### DISCUSSION TITLE
Year of The Broken Desktop - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#15] [General]
# Year of The Broken Desktop - Alan Pope's blog

This morning I attempted to start work on my desktop PC and couldn’t. The screen is black, it doesn’t want to wake up the displays. I used the old REISUB trick to restart, and it boots, but there’s no output on the display. I did some investigation and this post is mainly to capture my notes and so others can see the problem and perhaps debug and fix it.
The setup is an Intel Skull Canyon NUC connected to an external GPU enclosure which contains an NVIDIA GeForce RTX 2060.

https://popey.com/blog/2023/01/year-of-the-broken-desktop/

**Comment by tseliot** [comment]
Hey, sorry about the problems you faced with ubuntu-drivers. While we have had a fix for that specific bug since November in our git repository, I didn't really have the time to backport it to the stable Ubuntu releases. I should be able to fix this for good this month.

**Reply by popey** [reply]
Excellent news! Thanks!

**Reply by sgtnasty** [reply]
Good luck with NVIDIA on Linux, you are only asking for trouble.

**Reply by popey** [reply]
I've been using NVIDIA cards for years on Linux. It's no worse or better than Intel and AMD in my experience.
